RevenueCatUI/Templates/Components/CornerBorder.swift

```diff
@@ -0,0 +1,301 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CornerBorder.swift
+//
+//  Created by Josh Holtz on 9/30/24.
+
+import Foundation
+import SwiftUI
+
+#if PAYWALL_COMPONENTS
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct CornerBorderModifier: ViewModifier {
+
+    struct BorderInfo {
+
+        let color: Color
+        let width: CGFloat
+
+        init(color: Color, width: Double) {
+            self.color = color
+            self.width = width
+        }
+
+    }
+
+    struct RaidusInfo {
```

Comment:
Typo 🙂 
```diff
-    struct RaidusInfo {
+    struct RadiusInfo {
```

Reply:
This is how I leave my mark everywhere I go but... I'll change it just for you 🫶

Reply:
Haha I'm honored ❤️